src/bin/nodes/CommandFind.ts

```diff
@@ -71,7 +71,7 @@ class CommandFind extends CommandPolykey {
             result.port as Port,
           )}`;
         } catch (err) {
-          if (!(err instanceof nodesErrors.ErrorNodeGraphNodeNotFound))
+          if (!(err instanceof nodesErrors.ErrorNodeGraphNodeIdNotFound))
             throw err;
```

Comment:
Need to change this to `{ }`. Thinking I'll add in a eslint rule for `curly` here.

Reply:
This is done now, the `curly` rule is built into it and will autofix these things. There weren't that many places.